README.md

```diff
@@ -12,3 +12,29 @@ $ npm run storybook
 # Compiles and minifies for production
 $ npm run build-storybook
 ```
+
+## Style root color components
+
+```css
+:root {
+  --primary: #491e8c;
+  --secondary: #3b1873;
+  --dark-purple: #210e41;
+  --very-dark-purple: #0e061c;
+  --light-purple: #7e61ae;
+  --very-light-purple: #bdafd6;
+  --light-grey: #f2f0f7;
+  --dark-grey: #3c3940;
+  --Black: #000000;
+  --White: #ffffff;
+
+  --Error: #f44336;
+  --Success: #4bb543;
+  --Info: #1e88e5;
+
+  --Sale: #2e1359;
```

Comment:
Do we really need these 4 variables?
This is related to the ATD and not the components. I'm not sure we are even using them in any components

Reply:
Sorry, I forget to remove them.